src/components/RenderBooks.js

```diff
@@ -0,0 +1,38 @@
+import '../styles/RenderBooks.css';
+
+function RenderBooks() {
```

Comment:
- [ ] Kindly create a list containing individual books as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v3_0.md#:~:text=Contains%20a%20list%20to%20renders%20individual%20books).